Dockerfile

```diff
@@ -1,23 +1,26 @@
 # https://hub.docker.com/r/certbot/certbot
 # https://hub.docker.com/r/certbot/dns-cloudflare
-ARG ARCH=amd64
-FROM certbot/dns-cloudflare:${ARCH}-v1.30.0
+ARG TARGETARCH
```

Comment:
Actually we don't need this, I think that overrides the buildit defaults to empty.